composer.json

```diff
@@ -12,10 +12,10 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
```

Comment:
I've found a problem.
1. ubuntu 12.04 default php version is 5.3
2. ansible uses ```sudo``` internally for all steps
3. when you use [phpenv](https://github.com/CHH/phpenv) as travis does, ```php``` and ```sudo php``` is different versions of php
4. ```sudo composer``` can't satisfy ">=5.4" requirement, becase uses system php, which 5.3 
5. ansible wipe out all stderr from composer so it fails silently =/

Moreover, vagrant box is broken again, as there only one php version - 5.3

I'll fix this issue.